src/chording.rs

```diff
@@ -0,0 +1,288 @@
+//! Provides chord support for emulating a single layout event
+//! from multiple key presses. The single event press is triggered
+//! once all the keys of the chord have been pressed and the chord
+//! is released once all of the keys of the chord have been released.
```

Comment:
The first line should be a short description, as it is used in module description.